examples/PhasorDynamics/Tiny/TwoBus/Ieeet1/TwoBusIeeet1.cpp

```diff
@@ -0,0 +1,337 @@
+/**
+ * @file TwoBusIeeet1.cpp
+ * @author Luke Lowery (lukel@tamu.edu)
+ * @author Adam Birchfield (abirchfield@tamu.edu)
+ * @brief Example running a 2-bus system with exciter and governor
+ *
+ */
+#include "TwoBusIeeet1.hpp"
+
+#include <ctime>
+#include <iostream>
+
+#include <Model/PhasorDynamics/ComponentLibrary.hpp>
+#include <Model/PhasorDynamics/SystemModel.hpp>
+#include <Model/PhasorDynamics/SystemModelData.hpp>
+#include <Solver/Dynamic/Ida.hpp>
+#include <Utilities/Testing.hpp>
+
+// Temp, remove
+#include <Model/PhasorDynamics/Exciter/IEEET1/Ieeet1Data.hpp>
+
+int main()
+{
+  using namespace GridKit::PhasorDynamics;
+  using namespace AnalysisManager::Sundials;
+
+  using scalar_type = double;
+  using real_type   = double;
+  using index_type  = size_t;
+
+  using BusType      = BusData<scalar_type, index_type>::BusType;
+  using signal_type  = SignalNode<scalar_type, index_type>;
+  using machine_type = Genrou<scalar_type, index_type>;
+  using gov_type     = Governor::Tgov1<scalar_type, index_type>;
+  using exc_type     = Exciter::Ieeet1<scalar_type, index_type>;
+
+  std::cout << "Example: TwoBusTgov1 + IEEET1 Exciter\n";
+
+  // Model Data
+  SystemModelData<scalar_type, index_type> data;
+
+  // Set bus data
+  data.bus.resize(2);
+
+  data.bus[0].bus_id   = 0;
+  data.bus[0].bus_type = BusType::DEFAULT;
+  data.bus[0].Vr0      = 0.9949877346411762;
+  data.bus[0].Vi0      = 0.09999703952427966;
+
+  data.bus[1].bus_id   = 1;
+  data.bus[1].bus_type = BusType::SLACK;
+  data.bus[1].Vr0      = 1.0;
+  data.bus[1].Vi0      = 0.0;
+
+  // Set signal nodes data
+  data.signal.resize(2);
+
+  data.signal[0].name      = "omega";
+  data.signal[0].signal_id = 0;
+
+  data.signal[1].name      = "Pm";
+  data.signal[1].signal_id = 1;
+
+  data.signal[1].name      = "Efd";
+  data.signal[1].signal_id = 2;
```

Comment:
```suggestion
  // Set signal nodes data
  data.signal.resize(3);

  data.signal[0].name      = "omega";
  data.signal[0].signal_id = 0;

  data.signal[1].name      = "Pm";
  data.signal[1].signal_id = 1;

  data.signal[2].name      = "Efd";
  data.signal[2].signal_id = 2;
```
this fixes a heap buffer overflow

Reply:
Thank you. I did not catch this resizing!